Coverage takes some time and can't process pyx files.
So just run tests without it.

--- lxml-4.2.5/Makefile.1	2018-09-19 11:38:23.198889030 +0000
+++ lxml-4.2.5/Makefile	2018-09-19 11:38:47.067562249 +0000
@@ -68,10 +68,10 @@
 	$(PYTHON) test.py $(TESTFLAGS) $(TESTOPTS)
 
 test_inplace: inplace
-	$(PYTHON) test.py $(TESTFLAGS) $(TESTOPTS) $(CYTHON_WITH_COVERAGE)
+	$(PYTHON) test.py $(TESTFLAGS) $(TESTOPTS)
 
 test_inplace3: inplace3
-	$(PYTHON3) test.py $(TESTFLAGS) $(TESTOPTS) $(CYTHON3_WITH_COVERAGE)
+	$(PYTHON3) test.py $(TESTFLAGS) $(TESTOPTS)
 
 valgrind_test_inplace: inplace
 	valgrind --tool=memcheck --leak-check=full --num-callers=30 --suppressions=valgrind-python.supp \
